src/SourceCode.Clay.Tests/EnumConvertTests.cs

```diff
@@ -95,15 +99,22 @@ public static void EnumConvert_NonEnum()
             Assert.Throws<ArgumentOutOfRangeException>("value", () => EnumConvert.ToByte(0));
         }
 
-        [Fact(DisplayName = nameof(EnumConvert_ToEnumCheckedOfByte))]
-        public static void EnumConvert_ToEnumCheckedOfByte()
-        {
-            Assert.Equal(ByteEnum.MinValue, EnumConvert.ToEnumChecked<ByteEnum>(byte.MinValue));
-            Assert.Equal(ByteEnum.MaxValue, EnumConvert.ToEnumChecked<ByteEnum>(byte.MaxValue));
+        // TODO: Negative tests should pass
```

Comment:
I have logged a bug for @jcdickinson

Reply:
This should only throw for Checked, the following code should compile just fine in C#: `(EmptyEnum)50`.

Sometimes this is used for extensible enums (e.g. ETW does this).